src/screens/HomeScreen/HowYouFeel.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  View,
+  AsyncStorage,
+  StyleSheet,
+  FlatList,
+  Image,
+  Dimensions,
+  TouchableOpacity,
+} from 'react-native';
+
+export default class HowYouFeel extends Component {
```

Comment:
Start using functional components. It's being difficult to handle Class components. Also, some  Component methods are deprecated.